pkgs/top-level/all-packages.nix

```diff
@@ -23086,6 +23086,8 @@ in
 
   xosview2 = callPackage ../tools/X11/xosview2 { };
 
+  xournalpp = callPackage ../applications/editors/xournalpp { };
```

Comment:
`xournal` lives in `pkgs/applications/graphics/xournal` - maybe we should stay in there too for consistency?

Reply:
Ah, yeah, that's a good idea - moved the `xournalpp` underneath `xournal`, and put it in the same subdirectory.